controller_manager_msgs/msg/HardwareComponentsState.msg

```diff
@@ -0,0 +1,6 @@
+string name
+string type
+string class_type
```

Comment:
I actually added it because we have already a `type`-field. We are actually using the same nomenclature here as in "ComponentParser" and "HardwareInfo" class.